pkg/skaffold/build/custom/dependencies_test.go

```diff
@@ -25,7 +25,40 @@ import (
 	"github.com/GoogleContainerTools/skaffold/testutil"
 )
 
-func TestGetDependencies(t *testing.T) {
+func TestGetDependenciesDockerfile(t *testing.T) {
+	tmpDir, cleanup := testutil.NewTempDir(t)
+	defer cleanup()
+
+	// Directory structure:
+	//   foo
+	//   bar
+	// - baz
+	//     file
+	//   Dockerfile
+	tmpDir.Write("foo", "")
+	tmpDir.Write("bar", "")
+	tmpDir.Mkdir("baz")
+	tmpDir.Write("baz/file", "")
+	tmpDir.Write("Dockerfile", "FROM scratch \n ARG file \n COPY $file baz/file .")
+
+	customArtifact := &latest.CustomArtifact{
+		Dependencies: &latest.CustomDependencies{
```

Comment:
Can you add a test where Target is also specified?

Reply:
Removed target, as mentioned in comment below!